chia/wallet/puzzles/cat_loader.py

```diff
@@ -2,3 +2,5 @@
 
 CAT_MOD = load_clvm("cat.clvm", package_or_requirement=__name__)
 LOCK_INNER_PUZZLE = load_clvm("lock.inner.puzzle.clvm", package_or_requirement=__name__)
+
+CAT_MOD_HASH = CAT_MOD.get_tree_hash()
```

Comment:
you don't seem to be using this anywhere. is that deliberate?

Reply:
oh that was a bad cherry pick

Reply:
Please re-review